wurst/event/DamageType.wurst

```diff
@@ -155,12 +156,8 @@ function handler() returns bool
 
 init
     onEnter() ->
-        GetTriggerUnit()..addAbility(DAMAGE_TYPE_CHECK_ID)..makeAbilityPermanent(DAMAGE_TYPE_CHECK_ID, true)
-
-    forUnitsInRect(bj_mapInitialPlayableArea) iter ->
```

Comment:
Why did you remove this?

Reply:
forUnitsInRect is automatically iterated in OnUnitEnterLeave package